pyproject.toml

```diff
@@ -52,174 +43,142 @@ classifiers = [
 annotated-types = ["annotated-types"]
 attrs = ["attrs"]
 brotli = ["brotli"]
 cli = ["jsbeautifier", "uvicorn"]
 cryptography = ["cryptography"]
-jinja = ["jinja2"]
-jsbeautifier = ["jsbeautifier"]
+jinja = ["jinja2>=3.1.2"]
 jwt = ["python-jose", "cryptography"]
-mako = ["mako"]
-minijinja = ["minijinja"]
+mako = ["mako>=1.2.4"]
+minijinja = ["minijinja>=1.0.0"]
 opentelemetry = ["opentelemetry-instrumentation-asgi"]
 piccolo = ["piccolo"]
 picologging = ["picologging"]
 prometheus = ["prometheus-client"]
-pydantic = ["pydantic", "pydantic-extra-types"]
-redis = ["redis"]
-sqlalchemy = ["advanced-alchemy"]
-standard = ["jinja2", "jsbeautifier", "uvicorn", "fast-query-parsers"]
+pydantic = ["pydantic[email]", "pydantic-extra-types"]
+redis = ["redis[hiredis]>=4.4.4"]
+sqlalchemy = ["advanced-alchemy==0.2.2"]
+standard = ["jinja2", "jsbeautifier", "uvicorn", "fast-query-parsers>=1.0.2"]
 structlog = ["structlog"]
 
-full = [
-    "annotated-types",
-    "attrs",
-    "brotli",
-    "cryptography",
-    "jinja2",
-    "jsbeautifier",
-    "mako",
-    "minijinja",
-    "opentelemetry-instrumentation-asgi",
-    "piccolo",
-    "picologging",
-    "prometheus-client",
-    "pydantic",
-    "pydantic-extra-types",
-    "python-jose",
-    "redis",
-    "structlog",
-    "uvicorn",
-    "advanced-alchemy",
-    "fast-query-parsers",
-]
+[project.group.dev.dependencies.exceptiongroup]
+version = "*"
+python = "<3.11"
 
-[tool.poetry.scripts]
-litestar = { callable = "litestar.__main__:run_cli" }
+[project.scripts]
+litestar = "litestar.__main__:run_cli"
+
+[project.urls]
+"Issue Tracker" = "https://github.com/litestar-org/litestar/issues?q=is%3Aissue+is%3Aopen+sort%3Aupdated-desc"
+Changelog = "https://github.com/litestar-org/litestar/releases/"
+Twitter = "https://twitter.com/LitestarAPI"
+Reddit = "https://www.reddit.com/r/LitestarAPI"
+Discord = "https://discord.gg/MmcwxztmQb"
+Blog = "https://blog.litestar.dev"
+homepage = "https://litestar.dev/"
+repository = "https://github.com/litestar-org/litestar"
+documentation = "https://docs.litestar.dev/"
+
+[tool.hatch.metadata]
+allow-direct-references = true
+
+[tool.hatch.build.targets.sdist]
+include = [
+    'docs/PYPI_README.md',
+    '/Makefile',
+    '/litestar',
+    '/tests',
+    '/CHANGELOG.rst',
+]
 
 [build-system]
-requires = ["poetry-core>=1.0.0"]
-build-backend = "poetry.core.masonry.api"
+build-backend = "hatchling.build"
+requires = ["hatchling"]
+
+[tool.pdm.scripts]
+lint = "pre-commit run --all-files"
+test = "pytest tests docs/examples"
+docs-serve = "sphinx-autobuild docs docs/_build/ -j auto --watch litestar --watch docs --watch tests --port 8002"
+ci = { composite = ["lint", "test"] }
+
+[tool.pdm.dev-dependencies]
+test = [
+    "pytest",
+    "pytest-asyncio",
+    "pytest-cov",
+    "pytest-lazy-fixture",
+    "pytest-mock",
+    "pytest-rerunfailures",
+    "pytest-timeout",
+    "pytest-xdist",
+    "time-machine",
+]
+docs = [
+    "sphinx>=7.1.2",
```

Comment:
This works? Doesn't our theme depend on Sphinx `<7`?

Reply:
It builds locally and above, so I am thinking so? 😅